src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.cs

```diff
@@ -17,8 +17,8 @@ namespace Microsoft.CodeAnalysis.CSharp.ExtractMethod
 {
     internal partial class CSharpMethodExtractor : MethodExtractor
     {
-        public CSharpMethodExtractor(CSharpSelectionResult result)
-            : base(result)
+        public CSharpMethodExtractor(CSharpSelectionResult result, bool extractLocalFunction = false)
```

Comment:
please don't make optional.  this is internal and we should examine and force all callers to think about this and pass the right value.